app/services/cache_helper.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/AbcSize
+# rubocop:disable Metrics/MethodLength
+# rubocop:disable Metrics/ModuleLength
```

Comment:
This might be a sign of something :rotating_light: You should probably break this up into multiple modules, and that might give you more room to place cache-related methods that you've put directly into graphql types.